src/components/editor/property/PanelResourceNavItem.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { setCurrentComponent } from 'actions/index'
+import { useDispatch, useSelector } from 'react-redux'
+import { displayResourceValidations } from 'selectors/errors'
+import { selectCurrentComponentKey } from 'selectors/index'
+
+const PanelResourceNavItem = (props) => {
+  const dispatch = useDispatch()
+
+  const classNames = ['list-group-item', 'list-group-item-action']
+  const currentComponentKey = useSelector((state) => selectCurrentComponentKey(state, props.resourceKey))
+  if (props.componentKey === currentComponentKey) classNames.push('active')
+  // Check or bullet
+  const prefixMark = props.hasValue ? '\u2713' : '\u2022'
```

Comment:
Could you add a comment about what these unicode values are? (i.e.: checkmark and bullet)

Reply:
Is L14 adequate or are you looking for something different?